stickytimelineview/src/main/res/layout/recycler_section_header.xml

```diff
@@ -17,15 +18,18 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <View
+    <LinearLayout
```

Comment:
This view need an ID for the LinearLayout.
please, use the `lin` prefix.

Reply:
Thanks :) I modified it!